src/issuance.rs

```diff
@@ -653,6 +671,54 @@ mod tests {
         )
     }
 
+    fn generate_identity_point_asset_base() -> AssetBase {
+        let identity_point =
+            (pallas::Point::generator() * -pallas::Scalar::one()) + pallas::Point::generator();
+        AssetBase::from_bytes(&identity_point.to_bytes()).unwrap()
+    }
+
+    fn identity_point_asset_base_test_params(
+        note1_value: u64,
+        note2_value: u64,
+        finalize: bool,
+    ) -> (
+        OsRng,
+        IssuanceAuthorizingKey,
+        IssuanceValidatingKey,
+        IssueAction,
+        IssueBundle<Unauthorized>,
+        [u8; 32],
+    ) {
+        let (mut rng, isk, ik, recipient, sighash) = setup_params();
+
+        let asset = generate_identity_point_asset_base();
+
+        let note1 = Note::new(
+            recipient,
+            NoteValue::from_raw(note1_value),
+            asset,
```

Comment:
Reduce lines of code by calling `generate_identity_point_asset_base()` (the new name) from here and the other note

Reply:
Done.